netlify.toml

```diff
@@ -0,0 +1,18 @@
+[build]
```

Comment:
if possible, can we re-write this to yaml?

Reply:
done

Reply:
i did rewrite it, but netlify only accepts toml :(